aspnet/mvc/views/forms/sample/final/Views/Demo/RegisterValidation.cshtml

```diff
@@ -0,0 +1,16 @@
+﻿@model RegisterViewModel
+
+<form asp-controller="Demo" asp-action="RegisterValidation" method="post">
+    <div asp-validation-summary="ValidationSummary.ModelOnly"></div>
+    Email:  <input asp-for="Email" /> <br>
+    <span asp-validation-for="Email"></span><br>
+    Password: <input asp-for="Password" /><br>
+    <span asp-validation-for="Password"></span><br>
+    <button type="submit">Register</button>
+</form>
+
+<script src="https://ajax.aspnetcdn.com/ajax/jquery/jquery-2.1.4.min.js"></script>
+<script src="https://ajax.aspnetcdn.com/ajax/jquery.validate/1.14.0/jquery.validate.min.js">
+</script>
+<script src="https://ajax.aspnetcdn.com/ajax/mvc/5.2.3/jquery.validate.unobtrusive.min.js">
```

Comment:
@blowdart makes fun of me when I don't use HTTPS and  when I add the following I now get mixed content
```
 public void ConfigureServices(IServiceCollection services)
 {
     services.AddMvc();
     services.Configure<Microsoft.AspNet.Mvc.MvcOptions>(options =>
     {
         options.Filters.Add(new Microsoft.AspNet.Mvc.RequireHttpsAttribute());
     });
 }
```

Also this is copy/paste from our template generated code.